back/boxtribute_server/business_logic/box_transfer/agreement/queries.py

```diff
@@ -21,6 +22,19 @@ def resolve_transfer_agreement(*_, id):
 def resolve_transfer_agreements(*_, states=None):
     # No state filter by default
     state_filter = TransferAgreement.state << states if states else True
-    return TransferAgreement.select().where(
-        agreement_organisation_filter_condition() & (state_filter)
+    return (
```

Comment:
I feel like this would solve my problems with the `transferAgreements` query. well done